src/js/setUpSite.ts

```diff
@@ -98,6 +99,7 @@ const setUpSite = async (): Promise<void> => {
   setUpSearch(markerGroup, citiesToMarkers, data, searchElement, sliders);
   setUpAllFilters(markerGroup, citiesToMarkers, data, searchElement, sliders);
   setUpFilterPopup(markerGroup, citiesToMarkers, data, searchElement, sliders);
+  setUpAlerts();
```

Comment:
Nit: I've been putting these up higher, like line 88, since they don't need any arguments to work.